src/egon/data/datasets/power_plants/__init__.py

```diff
@@ -45,7 +46,8 @@ def __init__(self, dependencies):
                 insert_hydro_biomass,
                  wind_onshore.insert,
                  pv_ground_mounted.insert,
```

Comment:
Please update the version number (e.g. to 0.0.2) . Otherwise the tasks will be skipped even if it did not run with the wind offshore turbines.

Reply:
Done.